images.py

```diff
@@ -80,7 +82,12 @@ def _server_query(url, q):
 
 
 def get_image_url(
-    name, hints=[], size="large", thumb=False, enclosing_session=None, cache_only=False
+    name: str,
+    hints: List = [],
```

Comment:
Might want a `*` in the parameter list before `hints`, demanding that all parameters from here on must be keyword parameters